client/src/redux/actions/observations.js

```diff
@@ -22,6 +29,18 @@ export const addObservation = (userSession, observation) => async dispatch => {
   });
 };
 
+export const deleteObservations = userSession => async dispatch => {
+  const obs = await getObservations(userSession);
+  if (obs) {
+    const obsArray = JSON.parse(obs);
+    const numOfObservations = obsArray.length - 1;
```

Comment:
```suggestion
    const numOfObservations = obsArray.length;
```

The length of `obsArray` is the number of observations you want to delete.

Reply:
I see the logic in the <= above, but since the array starts at index 0, don't I still want to subtract 1 here to get the correct number of observations?

Reply:
The array does start at index 0, but the length of the array is the correct number of observations.

For example:

```js
const array = [ 1, 2, 3 ]; // This array starts at index 0,
console.log(array.length); // but its length is 3, which is how many numbers are in the array
```

Make sense?

Reply:
Hehe yes thank you! Actually trying it out also showed me that you are correct.